app/views/concepts/_details.html.haml

```diff
@@ -25,7 +25,11 @@
     %tr
       %td Preferred Name
       %td
-        %p= @concept.prefLabel({:use_html => true}).html_safe
+        %p
+        - if @concept.prefLabel({:use_html => false}).to_s.empty?
```

Comment:
can be changed to` @concept.prefLabel.blank?`, to prevent the `nil` exception

Reply:
this code `@concept.prefLabel({:use_html => false}).to_s.empty?` will not throw a `nil` exception. 

`nil.to_s` is a valid call that will turn a nil into an empty string. `xxx`.to_s.empty? ` is generally used to check for either `nil` or `empty?` condition.